.gitignore

```diff
@@ -1,3 +1,4 @@
 bin
 *.import
 .import
+*.translation
```

Comment:
Oh, does Godot update your translation files as well every time? 😆 

I'm still wondering, why it does that.

Reply:
I think this happens when I alternate between main and another branch. Still, better forget about these as they should be regenerated by Godot.